cli.js

```diff
@@ -6,17 +6,7 @@
 
 /* eslint no-console:0 */
 
-let katex;
-try {
-    katex = require("./");
-} catch (e) {
-    console.error(
-        "KaTeX could not import, likely because dist/katex.js is missing.");
-    console.error("Please run 'yarn' and 'yarn build' before running");
-    console.error("cli.js from the KaTeX repository.");
-    console.error();
-    throw e;
-}
+const katex = require("./");
```

Comment:
Is `dist` going to be checked in now?

Reply:
@kevinbarabash Yes. This enables (i) using KaTeX and CLI out of the box after cloning (#894) and (ii) using the repo directly as a dependency in `package.json` (#1229). This is also a step toward #1411 (fonts are now checked in the main repo).

Reply:
I was thinking of checking them in the next release, but it seems CLI and website needs it, so I've checked v0.12.0 in.

Reply:
I don't think this is necessary for #1411.  jest, webpack, and flow have ways to defining module mappings so that name modules resolve to `/packages/foo` (or wherever).  Even though I filed #894, I'm not sure how big of a deal that is.  I assume the primary reason for someone to clone the repo is to do some development work of some sort.  We could maybe simplify the process a bit so that `yarn install` also builds things.

As for #1229, I forgot that you could install things using `khan/katex`.  Doing so seems like a bad idea since the resulting package.json contains `"katex": "khan/katex"` as a dependency which indicates that the dependency is non-versions.  I don't think we should bother supporting this type of installation.

Can you say a little more about the website needs `dict` checked in?

If we are checking in `dist` is the expectation that whenever a change is made to `src` that it should also be accompanied by a change to `dict`?  If so, I think it would be nice if we did the following:
- had a GitHub Action that commit files back to `dist` if there are any changes after the build step completes
- add an appropriate entry to .gitattributes so that files in `dist` are collapsed in code reviews so that they don't add extra noise

Reply:
@kevinbarabash #894 and #1229 are just extra benefits. The major motivation is include the release commit to `master` branch history.

> If we are checking in `dist` is the expectation that whenever a change is made to `src` that it should also be accompanied by a change to `dict`? 

No, `dist` will be only touched by the release process, i.e., semantic-release bot with #2011.

> add an appropriate entry to .gitattributes so that files in `dist` are collapsed in code reviews so that they don't add extra noise

I didn't know we can do that.

Reply:
Another way I can think of is not to check `dist` in at all. This would break installing KaTeX using Git tags, e.g., `"katex": "KaTeX/KaTeX#v0.12.0"` or `https://cdn.jsdelivr.net/gh/KaTeX/KaTeX@v0.12.0/dist/katex.min.js`.

Reply:
After careful consideration, I think it'd be better not to include `dist` in the release commit. Closing in favor of #2385.

Reply:
I had a look at #2385.  I didn't realize that `dist` was being checked into a detached branch.  That's much better than what I thought was going on (checking `dist` into `master`).

> I was thinking of checking them in the next release, but it seems CLI and website needs it, so I've checked v0.12.0 in.

Is this still true?